sql/core/src/test/scala/org/apache/spark/sql/execution/adaptive/AdaptiveQueryExecSuite.scala

```diff
@@ -69,6 +69,18 @@ class AdaptiveQueryExecSuite
 
   setupTestData()
 
+  protected override def beforeAll(): Unit = {
```

Comment:
nit: we can override `sparkConf` with `super.sparkConf.set(CLASSIC_SHUFFLE_DEPENDENCY_FILE_CLEANUP_ENABLED, false)` to have special config for this test suite